crates/iota-types/src/digests.rs

```diff
@@ -159,6 +159,9 @@ impl fmt::UpperHex for Digest {
 )]
 pub struct ChainIdentifier(pub(crate) CheckpointDigest);
 
+pub const MAINNET_CHAIN_IDENTIFIER_BASE58: &str = "4btiuiMPvEENsttpZC7CZ53DruC3MAgfznDbASZ7DR6S";
```

Comment:
Add a TODO/create an issue to update this

Reply:
Actually thought about that and listed the occurrences locally, there is more than just this one, will open an issue

Reply:
https://github.com/iotaledger/iota/issues/5484